src/sections/map/maps-component.tsx

```diff
@@ -143,5 +154,18 @@ export default function MapComponent({
     onStationSelect,
   ]);
 
-  return <div ref={mapRef} style={{ height: "86vh", width: "100%" }} />;
+  // Live timestamp 
+  return (
+    <div className="w-full">
+      {currentTime && (
```

Comment:
Again I don't think we need to use currentTime state to conditionally render the time. I think time will always be available